README.md

```diff
@@ -1,7 +1,7 @@
 # Sudofy
 <img src="https://user-images.githubusercontent.com/54077752/110249544-990cd680-7f87-11eb-8300-dc5da1b9fb58.png"/>
 
-A project for Google Solution Challenge - 2021 (`Health monitoring system`)
+A project the for Google Solution Challenge - 2021 (`Health monitoring system`)
```

Comment:
Hey @Kimosop-Wakanda, the grammar in this sentence doesn't flow so well, please correct that.